### PR TITLE
İETT timetable viewer fix

### PR DESCRIPTION
**Pull request by erkinalp**
İETT bus timetables are unfortunately uploaded as images. This PR works around that.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2636,6 +2636,21 @@ CSS
 
 ================================
 
+iett.istanbul
+
+CSS
+.LineTimeDuyuru > li:nth-child(1) {
+    display:none;
+}
+.LineTimeDuyuru::before {
```
[the remark this answers is not in the document]
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:2339]: Why removing the original one and replacing?

**Comment by Gusted** [comment]
> The background information with the notice is İETT auctioned off some trips of some routes (or all trips in some). İETT buses and private operators are subject to slightly different operational rules, and buses look different too. So, they colour-coded which trips are operated by themselves and which trips are privately operated in the timetable listing. The notice replacement is actually a temporary fix as İstanbul Municipality is going to fix the problem for good in a few months by integrating the private buses fully, removing the noticeable differences in appearance and operational rule

Yes, but this isn't for dark reader to fix. It's pure for fixed of colors and visibility not for some `misinformation`.

**Comment by Gusted** [comment]
Let is be 🤷🏽‍♀️. I hear when there are issues about this.

**Comment by erkinalp** [comment]
> Yes, but this isn't for dark reader to fix. It's pure for fixed of colors and visibility not for some `misinformation`.

In this case, *our fix* was making the legend notice inaccurate, so I fixed to prevent complaints from Istanbulite users.

**Comment by erkinalp** [comment]
> https://ekonomi.haber7.com/ekonomi/haber/3014747-istanbulda-otobus-tasimaciliginda-yeni-donem-mecliste-kabul-edildi

They have eventually solved the problem that caused colour coding in the first place. Soon, the appearance and operational rule differences are going to disappear and so is the colour coding of the buses and the trips, then I will send a new pull request to adapt to that.

**Review by Scutua — APPROVED** [review]

**Pull request merged**